main.go

```diff
@@ -1,5 +1,15 @@
-// servedir command start http server, serving files in current directory
-//       go get foxygo.at/servedir
+// Command servedir starts http server serving files from given
+// directory on specified or next free port.
+//
+//     go get foxygo.at/servedir
+//     servedir --help
+//     usage: servedir [-p <port>] [-a] [<dir>]
+//
+//     Simple HTTP server, serving files from given directory.
+//
+//       -a	listen on all interfaces not just localhost
+//       -p int
+//            port number (default: os chosen free port)
```

Comment:
Should say something like "<dir> defaults to current directory if not specified" too

Reply:
ok, done.